doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-fluent-api-con.adoc

```diff
@@ -0,0 +1,103 @@
+[id='bpmn-fluent-api-con_{context}']
+= BPMN Process fluent API for {CENTRAL} processes
+
+{PRODUCT} provides a BPMN process fluent API that enables you to create business processes using factories. You can also manually validate the business process that you created using process fluent API. The process fluent API is defined in `org.kie.api.fluent` package.
+
+Therefore, instead of using BPMN2 XML standard, you can use the process fluent API to create business processes in a few lines of code.
+
+[[bpmn-fluent-api-examples-ref]]
+== Example requests with the BPMN process fluent API
+
+The following is an example of BPMN process fluent API requests for basic interactions with a business process,
+ifdef::DM,PAM[]
+download the *{PRODUCT} {PRODUCT_VERSION_LONG} Source Distribution* from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Red Hat Customer Portal] and navigate to `~/{PRODUCT_FILE}-sources/src/droolsjbpm-knowledge-$VERSION/kie-api/src/main/java/org/kie/api/fluent`.
+endif::[]
+ifdef::DROOLS,JBPM,OP[]
+see the process fluent API source in https://github.com/kiegroup/droolsjbpm-knowledge/tree/master/kie-api/src/main/java/org/kie/api/fluent[GitHub].
+endif::[]
+
+Creating and interacting with {CENTRAL} business processes::
+Following is an example of a basic business process with a script task, an exception handler, and a variable:
++
+--
+.Example request to create and interact with a {CENTRAL} business process
+[source,java,subs="attributes+"]
+----
+Process process =
+                // Create process builder
+                factory.processBuilder(processId)
+                       // package and name
+                       .packageName("org.jbpm")
+                       .name("My process")
+                       // start node
+                       .startNode(1).name("Start").done()
+                       // Add variable of type string
+                       .variable(var("pepe", String.class))
+                       // Add exception handler
+                       .exceptionHandler(IllegalArgumentException.class, Dialect.JAVA, "System.out.println(\"Exception\");")
+                       // script node in Java language that prints "action"
+                       .actionNode(2).name("Action")
+                       .action(Dialect.JAVA,
+                               "System.out.println(\"Action\");").done()
+                       // end node
+                       .endNode(3).name("End").done()
+                       // connections
+                       .connection(1,
+                                   2)
+                       .connection(2,
+                                   3)
+                       .build();
+----
+--
++
+In this example, the static `createProcess()` method is called from the `RuleFlowProcessFactory` class. The `createProcess()` method creates a new process and returns the `RuleFlowProcessFactory` that is used to create the process.
```

Comment:
Please update this section to be aligned with the community one and the text above

Reply:
Done